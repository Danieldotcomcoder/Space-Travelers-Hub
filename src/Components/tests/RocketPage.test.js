import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Rocket from '../RocketPage';
import store from '../../Redux/configureStore';

it('Renders Rocket Page', () => {
  const rocket = renderer.create(
    <Provider store={store}><Rocket /></Provider>,
  ).toJSON();
  expect(rocket).toMatchSnapshot();
});
