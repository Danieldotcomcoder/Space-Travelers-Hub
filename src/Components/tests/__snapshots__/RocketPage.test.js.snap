// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Rocket Page 1`] = `
<div
  className="rocket-c"
/>
`;
